changelog.md

```diff
@@ -74,6 +74,19 @@ result = 3.toNonZeroInt()
     .flatMap { (it * 2).toStrictlyPositiveInt() }
 ```
 
+### Deprecated
+
+The collections declared as [data classes] will be converted to [classes] (or
+[inline classes] when possible) in version [4.3.0], which means that their
+`copy` function is deprecated and will be unavailable after their conversion
+(issue [#97]).
```

Comment:
- [ ] Add a Kotlin code block showing how to copy a `NotEmptyList` without using the `copy` function.